README.md

```diff
@@ -1,7 +1,13 @@
 # TypedStruct
 
 [![Build Status](https://travis-ci.com/ejpcmac/typed_struct.svg?branch=develop)](https://travis-ci.com/ejpcmac/typed_struct)
-[![hex.pm version](http://img.shields.io/hexpm/v/typed_struct.svg?style=flat)](https://hex.pm/packages/typed_struct)
+[![Module Version](https://img.shields.io/hexpm/v/typed_struct.svg)](https://hex.pm/packages/typed_struct)
+[![Hex Docs](https://img.shields.io/badge/hex-docs-lightgreen.svg)](https://hexdocs.pm/typed_struct/)
+[![Total Download](https://img.shields.io/hexpm/dt/typed_struct.svg)](https://hex.pm/packages/typed_struct)
+[![License](https://img.shields.io/hexpm/l/typed_struct.svg)](https://github.com/ejpcmac/typed_struct/blob/master/LICENSE.md)
+[![Last Updated](https://img.shields.io/github/last-commit/ejpcmac/typed_struct.svg)](https://github.com/ejpcmac/typed_struct/commits/master)
```

Comment:
This is not really meaningful, provided that the lib will be in red at some point since I don’t see much more new updates to do to it. I’m still akin to do changes if there are some new interesting things to bring, but they will probably be spaces in time. Whether a package is maintained is not related to whether it has had some recent commits. If there is nothing to add, then there are no commits.